### ISSUE TITLE
Major events/slots traffic tracking

### ISSUE THREAD

**Issue by ziplock1** [VATSIM-Radar/vatsim-radar#432]
### Your Vatsim CID

1475694

### What do you suggest?

I wanna suggest to add special feature to track traffic following any major event route (like CTP, CTL, World Flight, Cross VATRUS e.g) or aircrafts flying with assigned slots from VATRUS VSS or VATCAN Bookings

### Why do you need that?

That's would be wonderful feature for VATSIM Radar future as they will be focused on event traffic.

### Validation

- [X] I have searched for other requests and didn't found any alike
- [X] I have also checked roadmap, and this is not planned

**Comment by FlyforLive** [comment]
Yup, that would help both for pre-planning and that pilots don't burden the frequency to ask for a slot

**Comment by daniluk4000** [comment]
Basically done with filters. Impossible for slots for now
